db/migrate/20180528181001_create_transition_notes.rb

```diff
@@ -0,0 +1,13 @@
+class CreateTransitionNotes < ActiveRecord::Migration[5.1]
+  def change
+    create_table :transition_notes do |t|
+
+      t.timestamps
+      t.belongs_to :educator, index: true, foreign_key: true
+      t.belongs_to :student, index: true, foreign_key: true
+      t.text "text"
+      t.datetime "recorded_at"
+      t.boolean "is_restricted", default: false
```

Comment:
It seems like it might match the design more closely if this were one record for the restricted and non-restricted fields.  Then you could enforce a uniqueness on "one transition note per student" whereas here the database allows any number of these.

Or is the idea that multiple counselors would record multiple notes?

Reply:
Ooooh, I like that data structure you suggest. I think it's a little late to refactor in that direction though since I'm user-testing tomorrow.